sotodlib/tod_ops/fft_ops.py

```diff
@@ -1188,3 +1198,51 @@ def build_hpf_params_dict(
         filter_params = params_dict
     
     return filter_params
+
+def get_common_noise_params(
+    aman,
+    signal=None,
+    merge=False,
+    merge_name='common_mode',
+    **fit_args
+):
+    """
+    Fits noise model with white and 1/f noise to the PSD of common noise.
+    This can be applied to any 1D data with same size as aman.samps.
+
+    Args
+    ----
+    aman : AxisManager
+        Axis manager which has samps axis aligned with signal.
+    signal : nparray
+        Signal sized nsamps or ndets x nsamps.
+        When signal is given in 2D shape, the common mode across detectors
+        is calculated with median method.
```

Comment:
It's also valid to pass in a precomputed common mode signal with shape 1 x nsamps. That option should be explained here in the docstring.